backend/contentBackend.js

```diff
@@ -266,3 +275,22 @@ export async function getSitePublicMeta() {
     homeRoute: await getHomeRoute(),
   };
 }
+
+/**
+ *
+ * @return {boolean} - returns if user is REACT_APP_VALID_HOSTED_DOMAIN matches their email
+ */
+/** @expose */
+export async function checkEmail() {
+  const profile = getCurrentUser(false);
```

Comment:
This entire function does nto need to exist. It's basically just `validate`. Even if it does need to exist, just move it to the other file at least.

Reply:
I will now be using validate